app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -121,6 +123,54 @@ class LoginActivity : AppCompatActivity(), ILoginView {
         logIn()
         cancelLogin()
         onEditorAction()
+        logInbuttonEnable()
+    }
+
+    private fun logInbuttonEnable() {
+        var emailEmpty  = true
+        var passwordEmpty = true
+        logIn.isEnabled = false
+        emailInput.addTextChangedListener(object: TextWatcher{
+            override fun afterTextChanged(s: Editable?) {
+
+            }
+
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {
+
+            }
+
+            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+                if(s.toString().trim()!=""){
```

Comment:
Add spaces around parentheses. Please reformat the code for all the changes that you have made.

Reply:
Sure.